contracts/strategy/yearn/YearnStrategy.sol

```diff
@@ -123,19 +123,13 @@ contract YearnStrategy is IStrategy, AccessControl, CustomErrors {
         onlyManager
     {
         if (amount == 0) revert StrategyAmountZero();
-        uint256 uninvestedUnderlying = _getUnderlyingBalance();
 
-        if (amount > uninvestedUnderlying) {
```

Comment:
this logic is removed because all of the funds are immediately invested into yearn in `invest()` function

Reply:
@brkomir I don't get this. The funds have to be withdrawn from the Yearn Vault

Reply:
actually the earlier code, while withdrawing was looking for if our strategy contract had any uninvested funds first then withdrawing from the yearn vault only the amount minus whatever uninvested lusd we had in the yearn strategy contract. But the uninvested lusd in our yearn strategy contract will always be zero since the vault contract automatically invests everything after sending lusd to our strategy contract. So he removed that part where we are checking for uninvested funds and directly withdraws whatever required directly from yearn vault.

Reply:
@gabrielpoca yep, as @dxganta said it, is not possible for us to end up with uninvested funds in the yearn strategy contract

Reply:
I see 👍